config/settings/umkc.py

```diff
@@ -1,7 +0,0 @@
-from .base import *
-
-# Custom settings for umkc deploy -- delete file for other deployments
-# -----------------------------------------------------------------------------
-DISPLAY_REFERRALS = False
-DISPLAY_APPOINTMENTS = False
-DISPLAY_CASEMANAGERS = False
```

Comment:
You shouldn't delete this file. Inherit local-umkc.py and production-umkc.py from both local.py and umkc.py, production.py and umkc.py